src/devices/Vl53L1X/DistanceMode.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Vl53L1X
+{
+    /// <summary>
+    /// The distance mode of the device
+    /// </summary>
+    public enum DistanceMode : byte
```

Comment:
From https://github.com/dotnet/iot/tree/main/src/devices/Vl53L0X, can we align on the naming? Seems similar to the "Precision" one.

Reply:
Renamed DistanceMode to Precision